config.json

```diff
@@ -902,6 +902,21 @@
       ]
     },
     {
+      "uuid": "7961c852-c87a-44b0-b152-efea3ac8555c",
+      "slug": "isbn-verifier",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 3,
```

Comment:
I would consider this a much, much easier difficulty. Perhaps put it after the RNA Transcription exercise?